bus-mapping/src/circuit_input_builder/execution.rs

```diff
@@ -940,6 +941,35 @@ impl EcAddOp {
         Self { p, q, r }
     }
 
+    /// Creates a new EcAdd op given input and output bytes from a precompile call.
```

Comment:
add comment to explain we expect the input bytes is valid (coordinate is valid Fp and they are in the curve)